go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.8.4
 	github.com/vmware-tanzu/net-operator-api v0.0.0-20210401185409-b0dc6c297707
-	github.com/vmware-tanzu/vm-operator/api v1.8.2
+	github.com/vmware-tanzu/vm-operator/api v1.8.3-0.20231114230806-852c1641447a
```

Comment:
Fine for now, but you might want to bump to a release tag before the next CAPV minor release